packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -19,6 +19,7 @@ import {
   fileURLToFilePath,
 } from '../utils/path.js';
 import { encodeInput, hasStatusCode } from './utils.js';
+import { createRequire } from 'node:module';
```

Comment:
This file is used in non-Node runtime, so we can't import it here. Any hacks to avoid it?